tests/derives.rs

```diff
@@ -0,0 +1,27 @@
+#[test]
+#[cfg(feature = "macros")]
```

Comment:
You don't need `#[cfg(feature = "macros")]` if you add the following to `./Cargo.toml`:

```toml
[[test]]
name = "derives"
required-features = [ "macros" ]
```

(It doesn't make much sense to run this test without the `macros` feature anyway.)